tests/query/v2/test_starts_ends_with.py

```diff
@@ -241,8 +388,105 @@ def test_ends_with(self):
                 [T_NULL_BAD_TYPE]
             ]
         }
+        self.check_out_of_order_result(resp, expected_data["rows"])
+
+    def test_not_ends_with(self):
+        stmt = "YIELD 'apple' NOT ENDS WITH 'le'"
+        resp = self.execute_query(stmt)
+        self.check_resp_succeeded(resp)
+
+        expected_data = {
+            "column_names" : [],
```

Comment:
The `column_names` not check. under ditto.

Reply:
current column_names ignores quotation marks when strings are used, so if we add the check right now it would cause tests to fail

Reply:
Add some comments for it.

Reply:
Or issue.

Reply:
Issue has been added: Incorrect expected data in CI test #355
Also comments can be found at line 77